core/src/main/java/nu/yona/server/messaging/service/FirebaseServiceException.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License, v.
+ * 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server.messaging.service;
+
+import java.io.Serializable;
+
+import nu.yona.server.exceptions.YonaException;
+
+public class FirebaseServiceException extends YonaException
+{
+	private static final long serialVersionUID = 8840294619248574727L;
+
+	private FirebaseServiceException(String messageId, Serializable... parameters)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "FirebaseServiceException" constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)